### PR TITLE
fixes #169 - created middle section of storelisting page

### LINKED ISSUE #169: Create middle section of store listing page
The store listing presently has a left side bar showing categories for navigation and a right side bar for similar apps suggetion. The next step is to create a middle section where the actual app details will be present. This will give the developers an opportunity to showcase the feature off their apps apart from their live app.

### Proposed design
The middle section will contain the following
* An app promo image, author's name and link, **one line** description of the app, app headline
* An image carousel showing app pictures
* app details which will include **getting started** with the app, **use of the app** and other relevant information which the developer will feel like sharing.

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #169. The middle section now shows app
details. The top portion will show basic information like
app name, promo image, app headline, one line app description.
Then there will be an image carousel showing apping images.
This will be followed by app details containing infomation like
getting started with the app, app use and other relevant information.
All these data is fetched from app's app.json file which will be
configured by the developer.

### Short description

**Changes:**

**Screenshots for the change:**

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link - http://loklakapps.surge.sh/details.html?q=whatsTrending

**Comment by djmgit** [comment]
The middle section supports mark down to html rendering, so to showcase the app, the developer has to create the corresponding .md files and mention them in app.json and the data will be rendered in the store listing page. As I mentioned before the every details about the configuration of app.json will be mentioned in the tutorial page on which I will be working next. Soon the store listing page will be integrated will all the apps after the app.json of all the apps are updated. (Working on it).

**Comment by mariobehling** [comment]
The preview pictures for the app are not scaled properly - in ratio - as mentioned several times elsewhere. An old error should not continue to persist, because it was there before. The time I am spending to explain this now is already several times that I would have needed to fix this. It does not make sense to collaborate in that way.

Please scale this picture in ratio and pay more attention to detail to make the development flow a better experience for everyone.

Do you really not see that the diagram is not a proper circle and that the table is squeezed as well?

Also ensure you use the complete width of the top center area, e.g. the red button should be further on the right.
screenshot from 2017-06-06 09-01-56

**Comment by djmgit** [comment]
@kavithaenair, @SKrPl, @Achint08 open for review

@mariobehling apolpgies for taking so long for addressing the issue you mentioned, I will be careful from the next time. I have tried to use up the entire width of the top section, scaled the top images and the preview images.

File: css/details.css [hunk 1]
```diff
@@ -17,15 +17,15 @@ body {
...
-
 .loklak-brand:hover {
     text-decoration: none;
     color: rgb(51,51,51);
 }
 
+.loklak-header {
+    color: #C8254D;
+}
+
```
Comment (kavithaenair) [inline css/details.css]: Why are the lines moved?

File: whatsTrending/getStarted.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#### whatsTrending app gives us a list of the top trending Hashtags on twitter.
+
+It provides us with statistics showing the top Hashtags and the corresponding count
+of the tweets containg them.
```
Comment (kavithaenair) [inline whatsTrending/getStarted.md]: Fix the typo

**Comment by djmgit** [comment]
@kavithaenair made the changes

File: css/details.css [hunk 3]
```diff
@@ -151,12 +148,126 @@ body {
     margin-left: 50px;
 }
 
+.app-data {
+    box-shadow: 0 -4px 8px 0 rgba(0,0,0,0.2);
```
Comment (Achint08) [inline css/details.css]: @djmgit Can you increase box-shadow for the bottom area?

**Comment by djmgit** [comment]
@Achint08 made the changes

**Comment by mariobehling** [comment]
Good job!

**Pull request merged**